README.md

```diff
@@ -11,44 +14,78 @@ Computations and statistics on manifolds with geometric structures.
 - The documentation of ```geomstats``` can be found on the [documentation website](https://geomstats.github.io/).
 - If you find ``geomstats`` useful, please kindly cite our [paper](https://arxiv.org/abs/1805.08308).
 
+## Install geomstats via pip3
 
-## Installation
-
-OS X & Linux:
+From a terminal (OS X & Linux), you can install geomstats and its requirements with ``pip3`` as follows::
 
 ```
 pip3 install -r requirements.txt
 pip3 install geomstats
 ```
 
-Pytorch and tensorflow requirements are optional, as geomstats can be used with numpy only.
+This installs the latest version uploaded on PyPi.
+
+## Install geomstats via Git
+
+From a terminal (OS X & Linux), you can install geomstats and its requirements via Git as follows::
 
-To change backend:
 ```
-export GEOMSTATS_BACKEND=pytorch
+pip3 install -r requirements
```

Comment:
I think you have to clone first, otherwise you don't have the file requirements. And it should be requirements.txt